### PR TITLE
fix(): fix long press for arrows in number input

### LINKED ISSUE #1174: Max file size - long click on arrows to be implemented
### Rationale

Requested by the Community Members. 
POS set up, Max File Size - the arrows ups and down need to be clicked multiple times; it would be better to _lock_ the arrows on longer click, to increase/decrease the field value. 
Screenshot 2023-02-28 at 22 31 17

https://discord.com/channels/623195163510046732/691259274646781962/1080031575921659957

### Prerequisites
n/a
### User story

Describe the most straightforward usage scenario

- [ ] I start setting up the POS
- [ ] I want to insert a bigger/smaller value in the Max file size using the arrows/buttons
- [ ] I click on the arrow and don't release the click until the number reaches the value I wanted to set.

### Acceptance criteria

Provide some steps that an end-user would be able to perform when the feature could be considered as implemented. E.g.:

- [ ] The basic scenario works as described
- [ ] If I do not enter a value, an error¹ is shown
- [ ] The value fits between the range of 2 - 100 GB.

### How to implement
`setTimeout` for the `onClick`?

### How to test

### PR DESCRIPTION
**Pull request by maparr**
It closes #1174

### CODE REVIEW

File: app/basicComponents/Input.tsx [hunk 1]
```diff
@@ -120,6 +120,8 @@ type Props = {
   maxLength?: any;
   autofocus?: boolean;
   iconLeft?: string;
+  min?: string | number;
+  max?: string | number;
```
Comment (brusherru) [inline app/basicComponents/Input.tsx]: Why not just
```suggestion
  min?: number;
  max?: number;
```
?

Comment (maparr) [reply app/basicComponents/Input.tsx]: it was a reference type to the original input, changed.

**Review by monikasmolarek — APPROVED** [review]
👍

**Review by brusherru — APPROVED** [review]
👍

**Pull request merged**